src/pages/contact.js

```diff
@@ -1,84 +1,164 @@
+                <h1>
+                  {twitterContentBlock.heading}{" "}
+                  <img src={twitterIcon} id="twitter-icon" alt="twitter icon" />
+                </h1>
+                <div>
+                  {documentToReactComponents(twitterContentBlock.content.json)}
                 </div>
-                <div className="pt-5">
-                    <h1>Join Us In-Person</h1>
-                    <div>OpenOakland meets every Tues,{' '}
-                        6:30pm at Oakland City Hall.
-                    </div>
-                    <a href="https://www.meetup.com/OpenOakland/events/" target="_blank" rel="noopener noreferrer">
-                        RSVP on Meetup <LaunchIcon/>
-                    </a>
+              </div>
+              <div className="pt-4">
+                <h1>{meetupContentBlock.heading}</h1>
+                <div>
+                  {documentToReactComponents(meetupContentBlock.content.json)}
+                  <a
+                    href="https://www.meetup.com/OpenOakland/events/"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    {meetupRsvpLink} <LaunchIcon />
+                  </a>
                 </div>
-
-                <div className="pt-5">
-                    <h1>Looking for data?</h1>
-                    <p><Link to="/about-data/">View OpenOUSD data and code.</Link></p>
-                    <div className="mt-3 strong">Check out other education-related OpenOakland projects:</div>
-                    <ul>
-                        <li><a href="https://www.opendisclosure.io/" target="_blank" rel="noopener noreferrer">Open Disclosure Oakland</a> - Campaign contributions for school board elections</li>
-                        <li><a href="https://trackg.org" target="_blank" rel="noopener noreferrer">trackg.org</a> - Measure G parcel tax spending</li>
-                    </ul>
+              </div>
+              <div className="pt-4">
+                <h1>{dataSourceContentBlock.heading}</h1>
+                <p>
+                  <Link to="/about-data/">{viewDataLink}</Link>
+                </p>
+                <div>
+                  {documentToReactComponents(
+                    dataSourceContentBlock.content.json
+                  )}
                 </div>
-            </Col>
-        </Row>
-        <Row>
-
+              </div>
+            </div>
+          </Col>
         </Row>
         <Row></Row>
-    </Container>
-
-  </Layout>
-)
+        <Row></Row>
+      </Container>
+    </Layout>
+  )
+}
 
 export default Contact
+
+// https://app.contentful.com/spaces/tqd0xcamk1ij/entries/41a8uyVvOiMFBBdqG2G6sU
+export const query = graphql`
+  query ContactPage($language: String) {
+    contentfulPage(slug: { eq: "contact" }, node_locale: { eq: $language }) {
+      title
+      content {
+        ... on ContentfulContactPageContent {
+          contactUs
+          signUpForUpdates
```

Comment:
I'm getting an error :/ 

`142:11  error  Cannot query field "signUpForUpdates" on type "ContentfulContactPageContent". `

Reply:
Weird! It was, and still is, working just fine for me (even though it looks like `signUpForUpdates` was unpopulated in Contentful). I just updated `signUpForUpdates` and the text did seem to update as expected. Would you mind restarting Gatsby and trying again & letting me know if the problem persists?

Reply:
Interesting...didn't work yesterday no matter how much I refreshed the data and restarted Gatsby. Works now!